docs/reference/installation.rst

```diff
@@ -84,7 +84,7 @@ If you want to use the API, you also need ``friendsofsymfony/rest-bundle`` and `
                         SonataNewsBundle: ~
 
 
-* Import the ``sonata_news.yml`` file and enable json type for doctrine:
+* Import the ``sonata_news.yml`` file :
```

Comment:
importing the file manually isn't needed for Symfony 4

You can remove the whole paragraph